utils/hare-shutdown

```diff
@@ -73,16 +74,17 @@ def stop_process(processes: Dict[str, List[utils.Process]]):
             utils.stop_parallel(local_procs)
 
 
-def shutdown_cluster():
+def shutdown_cluster(skip_consul_stop: bool):
     check_cluster()
     (processes, leader_node) = _get_processes_for_cluster()
-    stop_process(processes)
+    stop_process(processes, skip_consul_stop)
 
-    if leader_node and not utils.is_fake_leader_name(leader_node):
-        logging.info(f'Shutting down RC Leader at {leader_node}... ')
-        pkill: str = 'sudo pkill --exact -KILL consul &> /dev/null'
-        utils.exec_custom(utils.ssh_prefix(leader_node) + pkill,
-                          show_err=False)
+    if skip_consul_stop == False:
```

Comment:
As @knekrasov suggested, if we take out the consul shutdown code into a separate function, may be `stop_consul()` then we need not pass the flag to `shutdown_cluster()` in `main()` itself we can check `if skip_consul_stop` and invoke `stop_consul()` or not.